tests/testData/mockData.ts

```diff
@@ -2082,6 +2109,47 @@ export const invalidDurationProducts: MultiProduct[] = [
         productDurationId: '.',
         productDurationUnits: 'week',
         productDurationUnitsId: 'product-duration-units-1',
+        productValidity: '',
+        productValidityId: '',
+    },
+];
+
+export const invalidDurationTypeProducts: MultiProduct[] = [
+    {
+        productName: 'valid duration type',
+        productNameId: 'multiple-product-name-input-0',
+        productPrice: '.',
+        productPriceId: '.',
+        productDuration: '66',
+        productDurationId: '.',
+        productDurationUnits: 'week',
+        productDurationUnitsId: 'product-duration-units-1',
+        productValidity: '',
+        productValidityId: '',
+    },
+    {
+        productName: 'no duration type',
+        productNameId: '.',
+        productPrice: '.',
+        productPriceId: '.',
+        productDuration: '0',
+        productDurationId: '.',
+        productDurationUnits: '',
+        productDurationUnitsId: 'product-duration-units-1',
+        productValidity: '',
+        productValidityId: '',
+    },
+    {
+        productName: 'valid duration',
+        productNameId: 'multiple-product-name-input-0',
+        productPrice: '.',
+        productPriceId: '.',
+        productDuration: '66',
+        productDurationId: '.',
+        productDurationUnits: 'week',
+        productDurationUnitsId: 'product-duration-units-1',
+        productValidity: '',
+        productValidityId: '',
```

Comment:
This appears to be identical to the `valid duration type` product - is it necessary?